configs/mapping.yaml

```diff
@@ -29,6 +29,15 @@ mappings:
 - field: geometry
   issue-type: WGS84 out of bounds of England
   description: Geometry must be in England
+- field: geometry
+  issue-type: invalid WKT
+  description: Geometry must be in Well-Known Text (WKT) format
+- field: geometry
+  issue-type: invalid coordinates
+  description: Geometry must use WGS84, OSGB or Mercator coordinates
+- field: geometry
+  issue-type: invalid geometry not fixable
```

Comment:
isn't there a hyphen in this issue type? 'invalid geometry - not fixable'
https://datasette.planning.data.gov.uk/digital-land/issue_type
Thought it might matter if this is a mapping

Reply:
Thanks for pointing this out. I will check other issue types too.